### ISSUE TITLE
[A11Y] Unable to scroll inside “Learn more”/“What is a web compatibility issue” popups using up/down arrow keys

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3478]
**Browser / Version**: Firefox Nightly 82.0a1 (2020-09-16), Chrome 85.0.4183.102
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Using "Tab" key, navigate to "Learn more about web compatibility" link.
5. Using "Enter" key, activate the popup.
6. Use "Up/Down" arrow keys. 
7. Observe behavior.

**Expected Behavior:**
Scroll up/down inside the popup is performed.

**Actual Behavior:**
Scrolling does not occur inside the popup.

**Notes:**
1. Screenshot attached.
2. The popup needs to be in focus in order for the scrolling to works.

![ScrollingInsideNotWorking](https://user-images.githubusercontent.com/12184325/93483861-f2b21380-f909-11ea-870d-9c6266d002d8.gif)

**Watchers:**
@softvision-oana-arbuzov
@cipriansv
